test/integration/driver.go

```diff
@@ -88,6 +90,8 @@ func init() {
 		"Docker hub")
 	flag.StringVarP(&tag, "tag", "t", "",
 		"Docker tag")
+	flag.StringVarP(&mixerTag, "mixerTag", "", "latest",
```

Comment:
can we put a fixed (sha-based) tag? the test is supposed to be hermetic, and updates to mixer can accidentally break our tests.